pngcanvas.py

```diff
@@ -47,6 +47,16 @@ def gradient_list(start, end, steps):
             for i in range(steps + 1)]
 
 
+def rgb2rgba(rgb):
+    """Take a row of RGB bytes, and convert to a row of RGBA bytes."""
+    rgba = []
```

Comment:
Oh, right. We can probably use `itertools.chain` with my suggestion above but it's up to you.